app/services/platform.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'faraday'
+
+# Methods related to connecting to other platform services
+module Platform
+  def self.connection
+    faraday = Faraday.new do |f|
+      f.response :raise_error
+      f.adapter Faraday.default_adapter # this must be the last middleware
+      f.ssl[:verify] = false
+    end
```

Comment:
You may test this by using `faraday.basic_auth 'foo', 'bar'` and using the public URL for the remediations API.